man/journald.conf.xml

```diff
@@ -348,7 +348,10 @@
         <literal>notice</literal> for <varname>MaxLevelKMsg=</varname>,
         <literal>info</literal> for <varname>MaxLevelConsole=</varname>,
         and <literal>emerg</literal> for
-        <varname>MaxLevelWall=</varname>.</para></listitem>
+        <varname>MaxLevelWall=</varname>. These settings may be
```

Comment:
I will update the patch. I have been away for some time, what is the work flow to update a patch? Force update?
Did CI really fail or has it been cancelled?